hyp3_autorift/process.py

```diff
@@ -145,14 +154,16 @@ def process(reference: str, secondary: str, polarization: str = 'hh', band: str
             scene_url = get_download_url(scene)
             download_file(scene_url, chunk_size=5242880)
 
+        polarization = get_s1_copol(reference)
```

Comment:
I'd move this initialization down immediately before it's used by `geometry.bounding_box`